lib/mixpanel/client.ex

```diff
@@ -18,19 +19,53 @@ defmodule Mixpanel.Client do
     GenServer.cast(:mixpanel_client, {:track, event, properties})
   end
 
+  def track(events) when is_list(events) do
+    GenServer.cast(:mixpanel_client, {:track, events})
+  end
+
+  def engage(event) when is_map(event) do
+    GenServer.cast(:mixpanel_client, {:engage, event})
+  end
+
+  def engage(events) when is_list(events) do
+    GenServer.cast(:mixpanel_client, {:engage, events})
+  end
+
   def handle_cast({:track, event, properties}, state)  do
     properties = Dict.put(properties, :token, state.token)
-    {:ok, json} = JSX.encode([event: event, properties: properties])
-    body = String.to_char_list("data=#{ :base64.encode(json) }")
+    {:ok, json} = JSX.encode(event: event, properties: properties)
```

Comment:
Does bulk posting a non-array work?
We should probably add some ex-vcr specs at this point.